tests/ai/test_ai_manager.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+
+import gi  # nopep8
+gi.require_version('Gst', '1.0')  # nopep8
+gi.require_version('GLib', '2.0')  # nopep8
+from gi.repository import Gst as gst  # nopep8
+from gi.repository import GLib  # nopep8
+
+import cv2
+import numpy as np
+import random
+import unittest
```

Comment:
Clean this up

Reply:
These classes are being implemented on the "private tests"